### PR TITLE
Codecleaning (Closes #49)

### LINKED ISSUE #49: Remove unused/old code
Can we remove some of the (old) classes?

In particular:

- internal.crypto.CryptoModule with tests
- the other classes in internal.crypto (not internal.crypto.ppcp)
- the classes in internal.misc
- internal.bluetooth.BleCompat

related to https://github.com/CoraLibre/CoraLibre-android-sdk/pull/50

### PR DESCRIPTION
**Pull request by haitrec**
I removed many files that we probably don't need anymore, and removed some methods and fields from the remaining code (in particular in AppConfigManager). I moved some of the remaining files into subpackages of _sdk.internal_ where I think they fit best.
I also moved some of the classfiles that are currently not in use, but might be useful in the future, into _internal.error.old_ and _internal.bluetooth.old_. In the future we can move them into the respective packages (_internal.error_, _internal.bluetooth_) if they are still required, or delete them, if we do not need them for CoraLibre. For the bluetooth classes I would be glad if @theScrabi could check, if these should be deleted as well.

Before merging this PR I'd recommend to merge all ongoing work into master. Otherwise the many deletions and file movements in this PR might result in difficult/annoying rebasing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,78 +0,0 @@
-# Changelog for DP3T-SDK Android
-
-## Version 0.01 (2.6.2020)
-- Frok to get prestandard DP-3T to work like Google/Apple protocoll.
-- Rename to CoraLibre-android-sdk
```
Comment (Stypox) [inline CHANGELOG.md:5]: These three lines are relevant to coralibre: is it ok to remove them? I mean, changelog files are mostly useless if we will be using GitHub's releases, but anyway I am not completely sure if this should be removed

**Review by Stypox — CHANGES_REQUESTED** [review]
The things you removed look reasonable, so I think this is good.

**Review by theScrabi — APPROVED** [review]
LFTM :)

**Review by Stypox — APPROVED** [review]

**Comment by Stypox** [comment]
Ready for merge

**Pull request merged**